Microsoft.Toolkit.Uwp.SampleApp/SamplePages/MarkdownTextBlock/MarkdownTextBlockPage.xaml

```diff
@@ -0,0 +1,27 @@
+﻿<Page
+    x:Class="Microsoft.Toolkit.Uwp.SampleApp.SamplePages.MarkdownTextBlockPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:Microsoft.Toolkit.Uwp.SampleApp.SamplePages.MarkdownTextBlock"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"
+    mc:Ignorable="d">
+
+    <Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">        
+        <Grid.RowDefinitions>
+            <RowDefinition Height="auto"/>
+            <RowDefinition Height="*"/>
+            <RowDefinition Height="auto"/>
+            <RowDefinition Height="*"/>
+        </Grid.RowDefinitions>
+
+        <TextBlock Grid.Row="0" Text="Regular Text" Margin="12,12,12,0" FontSize="18"/>        
+        <TextBox x:Name="ui_unformattedText" Grid.Row="1" Margin="12" AcceptsReturn="True" TextWrapping="Wrap" />
```

Comment:
I actually tried that, but there was an issue where when I got the text back out of the TextBox it was missing some of the new lines required to keep the formatting of the markdown correct. I looked for a little while but couldn't figure it out, which is really confusing because this is exactly how I do it in Baconit, and how the old markdown sample app does it.